src/configuration/applier/hostescalation.cc

```diff
@@ -247,31 +216,39 @@ void applier::hostescalation::resolve_object(
   // Logging.
   logger(logging::dbg_config, logging::more) << "Resolving a host escalation.";
 
-  uint64_t host_id{get_host_id(*obj.hosts().begin())};
-  host_id_map::iterator it{engine::host::hosts_by_id.find(host_id)};
-  for (std::list<std::shared_ptr<engine::escalation> >::const_iterator
-           itt{it->second->get_escalations().begin()},
-       end{it->second->get_escalations().end()};
-       itt != end; ++itt) {
+  // Find host escalation
+  bool found{false};
+  std::string const& hostname{*obj.hosts().begin()};
+  auto p(engine::hostescalation::hostescalations.equal_range(hostname));
```

Comment:
you left an auto here ;)

Reply:
Yes, the type returned by equal_range is horrible :-), one line of 80 characters is not enough :-D